conformance/BUILD.bazel

```diff
@@ -1,9 +1,10 @@
 sh_test(
     name = "ct",
-    srcs = ["@com_google_cel_spec//tests:conftest.sh"],
+    srcs = ["@com_google_cel_spec//tests:conftest-nofail.sh"],
```

Comment:
Don't reuse the "ct" target for the dashboard.  We want the "ct" target to fail when a conformance test fails so we can guarantee that new changes don't break the select list of test files in this target.

Create a separate "ct_dashboard" target that a) uses the -nofail.sh script, and b) includes *all* of the simple textproto files, even those that are broken.  This way the dashboard shows the broken-ness of tests, while the "ct" target lets the developer gradually take on tests that they want to keep functional.

It would be nice if we could make the dashboard target "hidden" so that it doesn't get run when you do "bazel test ..." but will get run when you mention it explicitly.  But I can't find a bazel feature to do that.  We could abuse the test size description for this, but that seems inappropriate.